xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -1010,9 +1010,9 @@
         <Parameter Name="value" Type="System.Object" Index="1" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="index">The zero-based index at which the value should be inserted.</param>
+        <param name="value">The object to insert.</param>
+        <summary>Inserts an element into the collection at the specified index.</summary>
```

Comment:
Does this method throw if the index is beyond the limits of the list?

Reply:
Probably, but I can't tell you what it threw.  (This explicit implementation was deleted in netcoreapp2.0/netstandard2.0, when CollectionBase was brought back... which is true for the all of the cases where you had exception/behavior questions)